### PR TITLE
Adapt topic page #1215

### PR DESCRIPTION
**Pull request by marionb**
addresses https://github.com/openoereb/pyramid_oereb/issues/1215
TO TEST:
- use PR https://github.com/openoereb/pyramid_oereb/pull/1328
- turn off pdfA. For instructions see: #77

### CODE REVIEW

File: print-apps/oereb/topicTitle.jrxml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		<band height="20">
+			<textField textAdjust="StretchHeight" isBlankWhenNull="true" bookmarkLevel="1">
+				<reportElement stretchType="ElementGroupHeight" isPrintRepeatedValues="false" x="0" y="0" width="493" height="20" isPrintWhenDetailOverflows="true" uuid="16fcd742-e1b7-442f-8ca9-471cd50a0260">
+					<property name="com.jaspersoft.studio.unit.width" value="pixel"/>
+					<property name="com.jaspersoft.studio.unit.y" value="px"/>
+					<property name="com.jaspersoft.studio.unit.height" value="px"/>
+					<property name="com.jaspersoft.studio.unit.x" value="px"/>
+				</reportElement>
+				<textElement>
+					<font size="16" isBold="true"/>
```
Comment (vuilleumierc) [inline print-apps/oereb/topicTitle.jrxml]:
```suggestion
					<font size="15" isBold="true"/>
```

**Review by vuilleumierc — APPROVED** [review]
Looks good, thanks! Just one comment about the font size of the title

File: print-apps/oereb/topicTitle.jrxml (hunk 1 above)
Comment (voisardf) [reply print-apps/oereb/topicTitle.jrxml]: I was looking for that to be sure two title lines still do not fit in the reserved space 👍

**Review by voisardf — APPROVED** [review]
Looks good and worked for me. thanks

**Pull request merged**